readthedocs_theme/templates/includes/footer.html

```diff
@@ -28,6 +28,7 @@ <h4 class="ui sub header">Learn more</h4>
               <a class="item" href="https://docs.readthedocs.io/page/tutorial/index.html">Getting started guide</a>
               <a class="item" href="https://docs.readthedocs.io/page/config-file/index.html">Configure your project</a>
               <a class="item" href="/comparisons/github-pages/">Comparison to GitHub Pages</a>
+              <a class="item" href="/comparisons/gitbook/">Comparison to GitBook</a>
```

Comment:
If we do another one of these (Probably Netlify?), we should break out a simple comparison landing page `/comparisons/` that we link here. It can be a pretty simple index page that just lists our comparisions.

Reply:
👍🏼